config/initializers/0_clowder-config.rb

```diff
@@ -19,13 +19,25 @@
   redis_url = "#{config.dig('inMemoryDb', 'hostname')}:#{config.dig('inMemoryDb', 'port')}"
   redis_password = config.dig('inMemoryDb', 'password')
 
+  # Kafka
+  first_kafka_server_config = config.kafka.brokers[0]
+
+  kafka_server_config = {
+    brokers: "#{first_kafka_server_config.dig('hostname')}:#{first_kafka_server_config.dig('port')}",
+    security_protocol: first_kafka_server_config.dig('authtype')
```

Comment:
`authtype` would be `sasl` as per [Kafka provider example](https://redhatinsights.github.io/clowder/clowder/dev/providers/kafka.html#_json_structure). If there is a presence of cacert it should translate to `sasl_ssl`. This translation might happen always for `sasl` authtype, as Kafka would reject an access w/o SSL when SASL is used.

Reply:
I've refactored how I set the `security_protocol` . The three options from Clowder config are: 
- nothing (translates to `plaintext`)
- `sasl` (translate to `sasl_ssl`)
- `mtls` (raises an exception claiming that's unsupported)